flow-typed/Claim.js

```diff
@@ -82,7 +82,7 @@ declare type ChannelMetadata = GenericMetadata & {
 
```

Comment:
:small_red_triangle: I had a public collection (I think it was public) with only 1 item. I then edited it and removed that 1 entry. The GUI goes into a bad state. When I return to `/$/playlists`, I tile is in a bad state as well, so I can't undo changes.

Reply:
Does this still happen?

Reply:
Yes.  In this video, it also highlights some oddities when Watch Later is empty.

https://user-images.githubusercontent.com/64950861/181424040-5b3bee9a-87ad-40ff-ba34-bd6cdf8e2d3f.mp4

Reply:
Now it should be fixed?